setup.py

```diff
@@ -26,10 +26,10 @@
     packages=find_packages(),
     package_data={"fourmat": ("assets/*.*", "assets/.*")},
     install_requires=(
-        "click>=7",
-        "black==22.10.0",
-        "flake8-bugbear>=21,<22",
-        "flake8>=3,<4",
+        "click>=8",
```

Comment:
Upping the dependencies and maintaining version conventions for now--ie: pinned or ranged per existing convention